readux/books/templates/books/volume_export.html

```diff
@@ -287,6 +321,27 @@
       });
     </script>
 
+
+    <style>
```

Comment:
Is there a reason you included the CSS in the html and not in the external file.